.github/workflows/met-api-cd.yml

```diff
@@ -0,0 +1,68 @@
+name: MET API DEV CD
+
+
+on:
+  # Disable triggers for now.
+  # push:
+  #   branches:
+  #     - main
+  #   paths:
+  #     - "met-api/**"
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Environment (dev/test/prod)"
+        required: true
+        default: "dev"
+
+defaults:
+  run:
+    shell: bash
+    working-directory: ./met-api
+
+env:
+  APP_NAME: "met-api"
+  TAG_NAME: "dev"
+
+jobs:
+  forms-flow-bpm-cd-by-push:
+    runs-on: ubuntu-20.04
+
+    if: github.event_name == 'push' && github.repository == 'bcgov/met-public'
+    environment:
+      name: "dev"
```

Comment:
so do we plan to pass TEST as an arument?

Reply:
This would be the automated github action to deploy to dev, currently our Openshift has the triggers, but we can enable this any time if we decide github is a better place to have control over our CD (when we have a develop branch for example)

Reply:
yup,,we can control DEV, TEST , PROD as an trigger question for deployments..
STAGE can be parameter